include/mgp.py

```diff
@@ -1764,8 +1785,10 @@ def timestamp(self) -> int:
     def offset(self) -> int:
         """
         Supported stream sources:
-          - Kafka
+          * Kafka
 
+        Raises:
+            InvalidMessageError: If the message is not in a valid context.
         Raise InvalidArgumentError if the message is from an unsupported stream source.
```

Comment:
Should this be differently formatted, like:
Raises:
```suggestion
        InvalidArgumentError: if the message is from an unsupported stream source.
```

Reply:
Fixed